### PR TITLE
Validation: May 9th, Various alt-text added, and code-block indentation. 

### PR DESCRIPTION
**Pull request by Angelines-YG**
Validation fixes - alt-text added, and code-block indentation

### CODE REVIEW

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1.md [hunk 1]
```diff
@@ -25,37 +25,37 @@ The MVC Music Store is a tutorial application that introduces and explains step-
 
 The application we'll be building is a simple music store. There are three main parts to the application: shopping, checkout, and administration.
 
-![](mvc-music-store-part-1/_static/image1.jpg)
+![Screenshot of the ASP.Net Music Store overview menu, with options for selecting a genre or from the top picks selections.](mvc-music-store-part-1/_static/image1.jpg)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1.md:28]: "Screenshot of the A S P dot Net"

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1.md [hunk 2]
```diff
@@ -25,37 +25,37 @@ The MVC Music Store is a tutorial application that introduces and explains step-
...
 
 Visitors can browse Albums by Genre:
 
-![](mvc-music-store-part-1/_static/image2.jpg)
+![Screenshot of the ASP.NET music store genre albums selections menu that shows the collection of albums in a given genre.](mvc-music-store-part-1/_static/image2.jpg)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1.md:32]: "Screenshot of the A S P dot Net"

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1.md [hunk 3]
```diff
@@ -25,37 +25,37 @@ The MVC Music Store is a tutorial application that introduces and explains step-
...
 
 They can view a single album and add it to their cart:
 
-![](mvc-music-store-part-1/_static/image3.jpg)
+![Screenshot of the album selection window, showing the album's name, genre, artist, and price, with an option to add to cart.](mvc-music-store-part-1/_static/image3.jpg)
 
 They can review their cart, removing any items they no longer want:
 
-![](mvc-music-store-part-1/_static/image4.jpg)
+![Screenshot of the 'Review your cart' menu, with total price information and options to edit your cart or checkout. ](mvc-music-store-part-1/_static/image4.jpg)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1.md:40]: Remove quotes from "Review your cart". We should actually clarify this with Derek!

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-10.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ As shown previously, the ShoppingCart controller includes a CartSummary action m
 
 To create the CartSummary partial view, right-click on the Views/ShoppingCart folder and select Add View. Name the view CartSummary and check the "Create a partial view" checkbox as shown below.
 
-![](mvc-music-store-part-10/_static/image2.png)
+![Screenshot of the menu bar selections and options for creating a partial view of the shopping cart](mvc-music-store-part-10/_static/image2.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-10.md:38]: Add period to end of sentence.

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-10.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ This action returns a list of Genres which will be displayed by the partial view
 
 Right-click on the GenreMenu controller action and create a partial view named GenreMenu which is strongly typed using the Genre view data class as shown below.
 
-![](mvc-music-store-part-10/_static/image4.png)
+![Screenshot of the controller actions menu used for creating the partial view that lists the genre menu](mvc-music-store-part-10/_static/image4.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-10.md:66]: Add period to end of sentence.

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-10.md [hunk 6]
```diff
@@ -121,7 +121,7 @@ Finally, we'll need to update our Home Index view so that it can display a list
 
 Now when we run the application, we'll see our updated home page with top selling albums and our promotional message.
 
-![](mvc-music-store-part-10/_static/image1.jpg)
+![Music store homepage screenshot, showing the list of genres in a partial left view, the top picks albums at the bottom, and a large promotional message in the center of the page.](mvc-music-store-part-10/_static/image1.jpg)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-10.md:124]: "home page"?

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-2.md [hunk 7]
```diff
@@ -98,7 +98,7 @@ Run the project again and browse the following URLs:
 
 Accessing these URLs will invoke the action methods within our Controller and return string responses:
 
-![](mvc-music-store-part-2/_static/image5.png)
+![Screenshot of the window browser, showing an example of the action methods invoked when accessing the URL. It shows the following string response: Hello from Store.Details()](mvc-music-store-part-2/_static/image5.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-2.md:101]: Space out "u r l".

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-2.md [hunk 8]
```diff
@@ -120,7 +120,7 @@ ASP.NET MVC lets us easily do this without having to configure anything. ASP.NET
 
 Run the application and browse to /Store/Details/5:
 
-![](mvc-music-store-part-2/_static/image7.png)
+![Screenshot of the parameter being passed from the URL segment. The parameter is a string that reads Store.Details, ID=5.](mvc-music-store-part-2/_static/image7.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-2.md:123]: "u r l"

**Comment by Angelines-YG** [comment]
@Rick-Anderson 


The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**